app/controllers/api/v1/friends_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       render json: friends.errors, status: :bad_request
     end
   end
+
+  # GET /api/v1/friends/:id Show a specific friend GET request
+  def show
+    friend = Friend.find(params[:id])
+    if friend
+      render json: { data: friend }, status: :ok
```

Comment:
Either use this way you have here or use your private 'current_friend' private method.

Shouldn't have to mix the two ways you're grabbing the ID from the params.